src/runtime_src/core/pcie/tools/xbmgmt/xspi.cpp

```diff
@@ -2180,3 +2224,127 @@ int XSPI_Flasher::upgradeFirmware2Drv(std::istream& mcsStream0,
     // Disable bitstream guard.
     return removeBitstreamGuard(mDev.get(), mFlashDev, bsGuardAddr);
 }
+
+const std::string magic = "XRTDATA";
+struct flash_data_header {
+    char magic[7] = "";
+    char ver = 0;
+    uint32_t data_len = 0;
+    uint32_t parity = 0;
```

Comment:
As we have seen in the past with the axlf structures, here can be "future" needs to store additional metadata.   My I suggest adding 64 bytes of "reserved" future space.  For example:
```
struct flash_data_header {
  char magic[7];
  char verion = 0;
  uint32_t data_len_bytes;
  uint32_t parity;
  uint8_t reserved[16];
};
```

Reply:
will do.